resources/lang/hu/notifications.php

```diff
@@ -51,7 +51,7 @@
                 'beatmapset_discussion_post_new_empty' => 'Új poszt a ":title"-on :username:-tol',
                 'beatmapset_discussion_post_new_compact' => 'Új hozzászólás :username által: ":content"',
                 'beatmapset_discussion_post_new_compact_empty' => 'Új poszt :username által',
-                'beatmapset_discussion_review_new' => 'Új hozzászolás problémákat tartalmazó tartalomról :username által, ezen a beatmapen: ":title"  
+                'beatmapset_discussion_review_new' => 'Új hozzászolás problémákat tartalmazó tartalomról :username által, ezen a beatmapen: ":title"
```

Comment:
this should be fixed in crowdin instead (this change may be gone on next translation update)